static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -248,6 +253,43 @@ function useFetchGroupDetails(): FetchGroupDetailsState {
     refetch: refetchGroupCall,
   } = useGroup({groupId});
 
+  useEffect(() => {
```

Comment:
can we put a TODO to delete this entire useEffect when cleaning up streamline ui